modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2666,6 +2667,13 @@
 	end
 
 
+	function m.fastUpToDateCheck(prj)
+		if prj.fastuptodate == false then
+			m.element("DisableFastUpToDateCheck", nil, "true")
```

Comment:
Having another look at this, this should probably emit any time that `fastuptodate` is set.
```lua
if prj.fastuptodate ~= nil then
  m.element("DisableFastUpToDateCheck", nil, iif(prj.fastuptodate, "false", "true"))
end
```

The idea here is that, if the user is specifying the value then Premake should honour that request regardless of what the default may be - the default might change.

Reply:
Agree with @samsinsane, good catch. If you look at the other switches, "Default" always leaves the toolset's default behavior, while "On" and "Off" set it explicitly.

Reply:
I can definitely do this.  Sorry on the slow follow up, work + school has been insane recently.